src/Library/demos/Banner/main.blp

```diff
@@ -0,0 +1,46 @@
+using Gtk 4.0;
+using Adw 1;
+
+Gtk.Window window {
+  default-width: 800;
+  default-height: 600;
+  title: "Banner";
+  titlebar: Gtk.HeaderBar {
+    [end]
+    MenuButton button_menu {
+      icon-name: "open-menu-symbolic";
+      primary: true;
+    }
+  };
+
+  Adw.ToastOverlay overlay {
+    Adw.StatusPage {
+      title: "Adw Banner";
+      description: "Gnome Banner Test";
+
+      Box {
+        orientation: vertical;
+        halign: center;
+        valign: center;
+
+      Adw.Banner banner{
+        button-label: "Troubleshoot";
+        title: "An error occurred: Could not resolve host.";
+        revealed: true;
+        use-markup: true;
+        margin-bottom: 20;
+      }
```

Comment:
This is better, but the banner should really be up at the top of the window, with no margins, right under the header bar.

Then maybe make it so clicking "Troubleshoot" hides the banner and shows the toast. Where the banner is now, add a button so the banner can be re-opened.